arelle.pyw

```diff
@@ -1,14 +1,11 @@
 '''
```

Comment:
aribtrary line: Is it intentional to have the COPYRIGHT.md be empty? I figured it would have had actual information in it?

Reply:
Must have cut instead of copied, or something like that. Updated.